client/my-sites/plugins/plugins-discovery-page/index.jsx

```diff
@@ -89,6 +112,12 @@ const PluginsDiscoveryPage = ( props ) => {
 	} );
 
 	const isLoggedIn = useSelector( isUserLoggedIn );
+	const siteId = useSelector( getSelectedSiteId );
+	const sitePartnerBundle = useSelector( ( state ) =>
+		getSiteOption( state, siteId, 'site_partner_bundle' )
```

Comment:
If `getSiteOption( state, siteId, 'site_partner_bundle' )` is working do we also need to add it to the`sites/me` endpoint through the [Jetpack PR](https://github.com/Automattic/jetpack/pull/39905)?

Reply:
Unfortunately, I couldn't get the `getSiteOption( state, siteId, 'site_partner_bundle' )` to return anything without the [Jetpack PR](https://github.com/Automattic/jetpack/pull/39905).
From what I saw, options need to be whitelisted on the request and handled on the endpoint.